browser/src/control/Control.ColumnHeader.js

```diff
@@ -203,7 +209,13 @@ L.Control.ColumnHeader = L.Control.Header.extend({
 				width = 0;
 			}
 
-			width += dragDistance[0];
+			var isRTL = this.isCalcRTL();
+
+			if (isRTL)
+				width -= dragDistance[0];
+			else
+				width += dragDistance[0];
```

Comment:
Same. I think having braces around even single-line scopes is the convention used in .js/.ts.